### DISCUSSION TITLE
/room-assistant/entrypoint.sh: line 24: exec: room-assistant: not found

### DISCUSSION THREAD

**Discussion by leperroy** [mKeRix/room-assistant#857] [Q&A] [answered]
Hello,

I installed room-assistant version 2.18.4 on home assistant (core-2021.8.3, supervisor-2021.06.8, Home Assistant OS 6.2).

When I start it, it seems not to go on. The logs :

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[11:24:31] INFO: Setting up Home Assistant configuration
[11:24:31] INFO: Copying room-assistant configuration from /config/room_assistant
[11:24:31] INFO: Starting room-assistant
/room-assistant/entrypoint.sh: line 24: exec: room-assistant: not found
[cmd] /room-assistant/entrypoint.sh exited 127
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.

Any idea how to solve that ?

Thanks in advance

Guillaume

**Comment by mKeRix** [comment, marked as answer]
A recent update to the Home Assistant add-on base tooling broke the add-on temporarily without me noticing - sorry about that! This was also reported in mKeRix/hassio-repo#27. @wallacemreis was so nice to contribute a fix, which is currently building. If you re-install the add-on in about half an hour it should work again.

Re-installing leaves your config untouched by the way, so hopefully it shouldn't be too much of a hassle!

**Reply by leperroy** [reply]
Thanks a lot. It works. So nice to have an answer and a fix so fast. Have a great day.
